aggregate/housing_security/eviction_cases_housing_court.py

```diff
@@ -25,7 +25,7 @@ def load_filings():
 def eviction_cases(geography: str, write_to_internal_review=False):
     """Main Accessor"""
     filings = load_filings()
-    final = filings.groupby(geography).sum()["eviction_filings"]
+    final = filings.groupby(geography).sum()[["eviction_filings"]]
```

Comment:
the double brackets prevent `final` from being a series correct?

Reply:
yea exactly, if you select a single column from a df with double brackets it returns a single column dataframe instead of a series